### PR TITLE
Small notice for beginner in the tutorials page

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**
This pull request adds a small paragraph at the beginning of the documentation page about the tutorials. It aims to warn the beginners to read the User guide section...

**Dear reviewer**
This is a proposal...

### CODE REVIEW

**Review by AtreyeeS — DISMISSED** [review]
Thanks @bkhelifi !

**Review by QRemy — APPROVED** [review]
Thanks @bkhelifi,  this is a good advice !

File: examples/tutorials/README.rst [hunk 1]
```diff
@@ -6,6 +6,10 @@
 Tutorials
 =========
 
+**Notice for people not familiar with the TeV analysis technics:** it is
```
Comment (registerrier) [inline examples/tutorials/README.rst:9]:
```suggestion
**Notice :** it is
```

**Review by registerrier — COMMENTED** [review]
Thanks @bkhelifi . Good point. 

I would even be more general. It is not only an issue with TeV analysis in general, because there are also specific choices made in gammapy and everybody would profit in reading the user guide.

**Pull request merged**